canonical/ubuntu.com
# WD-12831 - rebrand /azure/sql

## Done

Rebrand /azure/sql. No content was changed, layout only

## QA

- [demo link](https://ubuntu-com-14159.demos.haus/azure/sql)
- [figma](https://www.figma.com/design/MinLlXouBnEDqaoAL3bTWC/24.10-Azure-bubble?node-id=35-3373&t=AnfmrwexpUiXbO00-0)
- View the site on demo
- Run through the following [QA steps](https://discourse.canonical.com/t/qa-steps/152)
- Check that the page is correct according to figma

## Issue / Card
[WD-12831](https://warthogs.atlassian.net/browse/WD-12831)

[WD-12831]: https://warthogs.atlassian.net/browse/WD-12831?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
